### PR TITLE
Modern theme for split view component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme)

File: src/app/public/modules/split-view/split-view-workspace.component.scss [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+  background: #ffffff;
   height: 100%;
   display: flex;
   flex-direction: column;
 }
+
+@include sky-theme-modern {
+
+  .sky-split-view-workspace {
+    border: 1px solid $sky-border-color-neutral-medium;
```
Comment (Blackbaud-ToddRoberts) [inline src/app/public/modules/split-view/split-view-workspace.component.scss]:
```suggestion
    border: 1px solid $sky-theme-modern-border-color-neutral-medium;
```

**Review by Blackbaud-ToddRoberts — CHANGES_REQUESTED** [review]
In responsive view, the "Back to list" button needs to use the sky icon font chevron in modern theme.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme)

File: src/app/public/modules/split-view/split-view-workspace.component.ts [hunk 2]
```diff
@@ -77,14 +78,16 @@ export class SkySplitViewWorkspaceComponent implements AfterViewInit, OnDestroy
...
 
-  public ngAfterViewInit(): void {
+  public ngOnInit(): void {
     this.splitViewService.isMobileStream
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe((mobile: boolean) => {
         this.isMobile = mobile;
         this.changeDetectorRef.markForCheck();
       });
+  }
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/split-view/split-view-workspace.component.ts:88]: ☝️ Not a part of the scope of this PR, but `isMobile` was throwing an Expression Changed error if you loaded the page starting out in a small window. Moving the check to `ngOnInit` fixes the error.

**Comment by Blackbaud-AlexKingman** [comment]
> In responsive view, the "Back to list" button needs to use the sky icon font chevron in modern theme.

@Blackbaud-ToddRoberts Done.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-modern-theme)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**